OpenEdXMobile/src/main/java/org/edx/mobile/http/cache/CacheManager.java

```diff
@@ -12,71 +12,99 @@
 import org.edx.mobile.util.Sha1Util;
 
 import java.io.File;
-import java.io.FileInputStream;
-import java.io.FileOutputStream;
 import java.io.IOException;
 import java.nio.charset.Charset;
 
+/**
+ * The cache manager for HTTP responses. The cache is stored on the filesystem, within the
+ * application directory.
+ *
+ * @deprecated This is deprecated since with the transition to OkHttp, we're now relying on it's own
+ * automatic caching mechanism. However, this is still being kept around as a read-only resource for
+ * a while, being queried as a last resort by as a last resort by the CustomCacheQueryInterceptor,
+ * to facilitate the transition without any user-facing issues. This may be removed later after a
+ * significant percentage of the user base have upgraded to a version that uses the OkHttp API in
+ * the Courseware module.
```

Comment:
@BenjiLee: OkHttp has it's own built-in cache manager that we would be using from now on. However, when transitioning the Courseware module to OkHttp, the previous custom cache that we were using won't be available to OkHttp, which might impact user experience. Thus, I'm keeping around the old cache manager for now, and it would be queried by a custom interceptor if OkHttp doesn't find anything in the cache. After most of the user base has updated the app to the version that uses OkHttp, we can remove this legacy cache on the assumption that the vast majority of the data would them have been cached on the new format.

@miankhalid: The repetition has been fixed in commit 8ed033c05bc23ed0c03c40eee901913a428309f8.